packages/http-client/src/index.ts

```diff
@@ -21,6 +21,7 @@ export type Data = {
     projectId: string;
     version: number;
     createdAt: string;
+    updatedAt: string;
     pages: Pages;
     breakpoints: [Breakpoint["id"], Breakpoint][];
```

Comment:
please use Array<> syntax, much easier to read here

Reply:
> updatedAt: string;
my code is just string ;-)

Reply:
true, just read that other piece and was confused